about.html

```diff
@@ -0,0 +1,53 @@
+---
+layout: page
+---
+<div id="about">

```

Comment:
Instead of wrapping `<header>` in `<div>`, assign an ID or class name to `<header>` itself.